packages/compiler/core/checker.ts

```diff
@@ -2216,34 +2183,6 @@ export function createChecker(program: Program): Checker {
     return clone;
   }
 
-  /**
-   * useful utility function to debug the scopes produced by the binder,
-   * the result of symbol merging, and identifier resolution.
-   */
-  function dumpScope(scope = globalNamespaceNode, indent = 0) {
```

Comment:
I mean, maybe? I'll keep it if anyone ever used it, but I found it wasn't that helpful for me in the end and maintaining it might be annoying. I have plans to include this information in my "pretty" colorized output.